systemtest/src/test/java/io/strimzi/systemtest/security/SecurityST.java

```diff
@@ -1328,6 +1331,122 @@ void testOwnerReferenceOfCASecrets() {
+        Date initialKafkaBrokerCertStartTime = kafkaBrokerCert.getNotBefore();
+        Date initialKafkaBrokerCertEndTime = kafkaBrokerCert.getNotAfter();
+
+        LOGGER.info("Change of kafka validity and renewal days - reconciliation should start.");
+        CertificateAuthority newClusterCA = new CertificateAuthority();
+        newClusterCA.setRenewalDays(150);
+        newClusterCA.setValidityDays(200);
+        KafkaResource.replaceKafkaResource(clusterName, k -> k.getSpec().setClusterCa(newClusterCA));
+
+        // Wait for reconciliation and verify certs have been updated
+        StatefulSetUtils.waitTillSsHasRolled(KafkaResources.kafkaStatefulSetName(clusterName), 3, kafkaPods);
+
+        // Read renewed secret/certs again
+        clusterCASecret = kubeClient().getSecret(clusterName + "-cluster-ca-cert");
+        cacert = SecretUtils.getCertificateFromSecret(clusterCASecret, "ca.crt");
+        Date changedCertStartTime = cacert.getNotBefore();
+        Date changedCertEndTime = cacert.getNotAfter();
+
+        // Check renewed Broker kafka certificate dates
+        brokerCertCreationSecret = kubeClient().getSecret(clusterName + "-kafka-brokers");
+        kafkaBrokerCert = SecretUtils.getCertificateFromSecret(brokerCertCreationSecret, clusterName + "-kafka-0.crt");
+        Date changedKafkaBrokerCertStartTime = kafkaBrokerCert.getNotBefore();
+        Date changedKafkaBrokerCertEndTime = kafkaBrokerCert.getNotAfter();
+
+        LOGGER.info("Initial ClusterCA cert dates: " + initialCertStartTime + " --> " + initialCertEndTime);
+        LOGGER.info("Changed ClusterCA cert dates: " + changedCertStartTime + " --> " + changedCertEndTime);
+        LOGGER.info("KafkaBroker cert creation dates: " + initialKafkaBrokerCertStartTime + " --> " + initialKafkaBrokerCertEndTime);
+        LOGGER.info("KafkaBroker cert changed dates:  " + changedKafkaBrokerCertStartTime + " --> " + changedKafkaBrokerCertEndTime);
+
+        String msg = "Error: original cert-end date: '" + initialCertEndTime +
+                "' ends sooner than changed (prolonged) cert date '" + changedCertEndTime + "'!";
+        assertThat(msg, initialCertEndTime.compareTo(changedCertEndTime) < 0);
+        assertThat("Broker certificates were not renewed",
+                initialKafkaBrokerCertStartTime.compareTo(changedKafkaBrokerCertStartTime) < 0
+                        && initialKafkaBrokerCertEndTime.compareTo(changedKafkaBrokerCertEndTime) < 0);
+    }
+
+    @Test
+    void testClientsCACertRenew() {
+        KafkaResource.createAndWaitForReadiness(KafkaResource.kafkaEphemeral(clusterName, 3)
+            .editOrNewSpec()
+                .withNewClientsCa()
+                    .withRenewalDays(15)
+                    .withValidityDays(20)
+                .endClientsCa()
+            .endSpec()
+            .build());
+
+        String username = "strimzi-tls-user";
```

Comment:
Wouldn't be better to make some "generated" name like in other tests? Just nit

Reply:
Good idea